src/components/Button.js

```diff
@@ -1,22 +1,25 @@
 import React, { Component } from 'react';
-
 import PropTypes from "prop-types";
-import Big from "big.js";
 
 export default class Button extends Component {
```

Comment:
* [ ] convert this to functional component.
 `export default ButtonPanel (props)` or `export default ButtonPanel ({data, processInput})`